lib/project_types/theme/commands/push.rb

```diff
@@ -39,10 +39,13 @@ class Push < ShopifyCLI::Command::SubCommand
           flags[:ignores] ||= []
           flags[:ignores] |= pattern
         end
+        parser.on("-f", "--force") { flags[:force] = true }
       end
 
       def call(_args, name)
         root = root_value(options, name)
+        return unless theme_directory?(root) || current_directory_confirmed?
```

Comment:
/nit doing the check in your head of `unless` plus an `||` statement feels is a little hard to read